.travis.yml

```diff
@@ -1,3 +1,3 @@
 jdk: oraclejdk8
 language: scala
-script: sbt scripted
+script: sbt ^scripted
```

Comment:
What does `^` mean? I've never seen it before!

Reply:
`^` is _the same_ as `+`, but for `crossSbtVersions` instead of `crossScalaVersions`:

http://www.scala-sbt.org/1.0/docs/sbt-1.0-Release-Notes.html#sbt-cross-building

There is also `^^` which is equivalent to `++`.

Reply:
Thanks - it's hard to Google for `^`!